conda/mpich/conda_build_config.yaml

```diff
@@ -203,11 +225,11 @@ moose_sysroot:                                              # [linux]
   - sysroot_linux-64 2.17 h4a8ded7_13                       # [linux]
 
 moose_libcxx:                                               # [osx]
-  - libcxx 12.0.1 habf9029_0                                # [not arm64 and osx]
+  - libcxx 12.0.1 habf9029_1                                # [not arm64 and osx]
   - libcxx 12.0.1 h168391b_0                                # [arm64]
 
 moose_libclang:                                             # [osx]
-  - libclang 12.0.1 default_he082bbe_4                      # [not arm64 and osx]
+  - libclang 13.0.0 default_he082bbe_0                      # [not arm64 and osx]
```

Comment:
I know this is strange, but this is what pyqt/qt _demands_. And it seems to not conflict with our 12.x requirements.

Reply:
I seem to remember always having to update `libclang` with the main clang version we use in the package. I find it interesting that you didn't need to adjust `libcxx` as well.